Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.cpp

```diff
@@ -246,7 +246,7 @@ namespace AzToolsFramework
 
         if (eventType == QEvent::Type::MouseMove)
```

Comment:
could this be cleaned up by moving these cases into a switch block

Reply:
I think it makes sense for that change to happen as a separate PR but it's not a bad idea for sure